samza-core/src/main/scala/org/apache/samza/util/CoordinatorStreamUtil.scala

```diff
@@ -33,24 +32,21 @@ import org.apache.samza.system.{StreamSpec, SystemAdmin, SystemFactory, SystemSt
 import org.apache.samza.util.ScalaJavaUtil.JavaOptionals
 
 import scala.collection.JavaConverters._
-import scala.collection.immutable.Map
 
 object CoordinatorStreamUtil extends Logging {
+  val COORDINATOR_STREAM_FACTORY = "coordinatorstreamFactory"
```

Comment:
The config needs to be comply with our existing naming conventions, which is separated using dot, e.g. job.container.count. So I would suggest something like job.coordinatorstream.config.factory